portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -37,6 +37,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     BlobstoreService blobstoreService =
         BlobstoreServiceFactory.getBlobstoreService();
     String uploadUrl = blobstoreService.createUploadUrl("/data");
+    System.out.println("The upload url is "+uploadUrl);
```

Comment:
Should have blank in both sides of "+":
"The upload url is " + uploadUrl

ClangTidy should fix it.